indico/htdocs/js/indico/modules/timetable/timetable/Layout.js

```diff
@@ -402,7 +401,7 @@ type("CompactLayoutManager", ["IncrementalLayoutManager"],
                  algData.grid.push([startMin/60%24, algData.topPx]);
              }
 
-             if (!algData.active) {
+             if (algData.active == 0) {
```

Comment:
why this change? if it's bool-ish we actually want `!`; and if you want to compare against the integer it should be `===`

Reply:
it is not bool-ish :P the name is misleading, this variable is kind of a counter